vanilla-framework/vanilla-framework
# Add latest run script

This builds on @anthonydillon's work in #1076.

I've added some additions:

- Run on a specific port, `8101`
- Use the very latest run stuff
- Update Gemfile.lock
- `clean` should delete `.bundle` as well
- Ignore unneeded files from jekyll build
- Expand README - to suggest `./run build` before `./run serve` (people are probably more interested in building)

QA
--

- Pull this branch
- Run `./run clean` and check it removes all the build artefacts
- Run `./run` and check it sets up the project in a container with all its dependencies and the site works
- Run `./run watch` and make a change to the Sass and check after a refresh the change is visible
- Run `./run test` and check that runs the tests
